tests/testconnmngr.nim

```diff
@@ -96,7 +96,7 @@ suite "Connection Manager":
     await connMngr.close()
 
   asyncTest "get conn with direction":
-    let connMngr = ConnManager.new()
+    let connMngr = ConnManager.new(maxConnsPerPeer = 2)
```

Comment:
Not sure if it's a good idea to add this at the moment. Even if we lower this value to 1, the test will still pass.